### DISCUSSION TITLE
[A-L] (2023/24) Foglio 4 - Esercizio 6

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#294] [Esercizi - Viaggi/Piazza]
![Screenshot 2023-10-28 110513](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/de9264b7-cf63-4bbd-8ed6-a06ba0e6e174)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/d2d63afe-a689-4ad3-8fa4-0acadd1fc3c3)

**Comment by Elia-Belli** [comment]
Analogo a @CiottoloMaggico 
Riscriviamo le equazioni nel formato cinese (anche se non vogliamo applicare il **TCR**): 
Moltiplichiamo ambo i lati della 1° eq per $[3]^{-1}\_{10}=[7]\_{10}$ : $x\equiv 28(10)\iff x\equiv 8(10) $
Moltiplichiamo ambo i lati della 2° eq per $[2]^{-1}\_{9}=[5]\_{9}$ : $x\equiv 35(9)\iff x\equiv 8(9)$
Moltiplichiamo ambo i lati della 3° eq per $[5]^{-1}\_{12}=[5]\_{12}$ : $x\equiv 5a(12)$


Adesso applico il metodo di sostituzione:
Riscrivo la prima eq come $(\star)\ x=8+10t_1$
Sostituisco alla seconda: $8+10t_1\equiv 8(9)\iff 10t_1\equiv 0(9)\iff t_1\equiv 0(9)\iff t_1=9t_2$
Sostituisco in $(\star)\ x=8+10\cdot (9t_2)$
Sostituisco alla terza: $8+10\cdot9t_2\equiv 5a(12)\iff 10\cdot 9t_2\equiv 5a-8(12)\iff 6t_2\equiv 5a-8(12)$ che ammette soluzione solo se $(6,12)=6|(5a-8)\iff 5a-8=6q$ che è equivalente a risolvere l'eq congruenziale $5a-8\equiv 0(6)$:
$$5a \equiv 8(6)\iff 5a\equiv 2(6) \iff a\equiv 10(6) \iff a\equiv 4(6) \Rightarrow a=4,10$$
Avendo trovato i valori di $a$ per cui il sistema è compatibile, risolviamolo:
- Se $a=4$ : $6t_2\equiv 20 -8(12)\iff 6t_2\equiv 0(12) \iff t_2\equiv 0(2)\iff t_2=0+2t_3$
$\Rightarrow (\star)\ x=8+10\cdot 9 (0+2t_3)\iff x= 8+ 10\cdot 9 \cdot 2t_3 \iff x\equiv 8(180)$
- Se $a=10$ : $6t_2\equiv 50 -8(12)\iff 6t_2\equiv 6(12) \iff t_2\equiv 1(2)\iff t_2=1+2t_3$
$\Rightarrow (\star)\ x=8+10\cdot 9 (1+2t_3)\iff x= 98+ 10\cdot 9 \cdot 2t_3 \iff x\equiv 98(180)$
